src/config/laravel-localization.php

```diff
@@ -88,6 +88,14 @@
          * LARAVEL_LOCALIZATION_LANG_DIRS=resources/lang,Modules/Blog/Resources/lang
          */
         'lang_dirs' => [resource_path('lang')],
+
+        /*
+         * You can customize the regexp for lang files to be able to exclude certain files.
+         */
+        'file_regexp'  => [
+            'php' => '/^.+\.php$/i',
```

Comment:
@kg-bot this is now redundant config ..

Reply:
ah cool .. thx for fixing this. I am still a bit of a newbie when it comes to Laravel